gdx-controllers-desktop/src/main/java/com/badlogic/gdx/controllers/desktop/support/JamepadController.java

```diff
@@ -31,7 +33,7 @@ public class JamepadController implements Controller {
 
     private final CompositeControllerListener compositeControllerListener = new CompositeControllerListener();
     private final IntMap<Boolean> buttonState = new IntMap<>();
-    private final IntMap<Float> axisState = new IntMap<>();
```

Comment:
Libgdx has IntFloatMap which you can use to achieve this

Reply:
Nice, that's what I was looking for. Thx!